frame_level_models.py

```diff
@@ -191,3 +193,48 @@ def create_model(self,
         model_input=activation,
         vocab_size=vocab_size,
         **unused_params)
+
+class FrameLevelLSTMLogisticModel(models.BaseModel):
+
+  def create_model(self, model_input, vocab_size, num_frames, **unused_params):
+    """Creates a model which uses a logistic classifier over the output
```

Comment:
Remove the specification of "logistic classifier" since it will be controllable through flags.